charts/pega/README.md

```diff
@@ -841,20 +841,20 @@ certificates:
 ```
 ## Deploying Hazelcast in Client Server Mode 
 
-The values.yaml section for `hazelcast` is used for configuration of a Client Server form of deployment for Hazelcast. Pega Platform supports the client-server model for the Hazelcast service, which provides cluster communication and distributes Pega Platform features across Kubernetes PODs. This deployment model introduces 
-independent scalability for both servers and clients in Pega Platform. **For Kubernetes based deployments Client server is the default and the recommended form of deployment.**
+The values.yaml section for `hazelcast` is used for configuration of a Client Server form of deployment for Hazelcast. 
+Pega Platform deployments using 8.6 and later, supports the client-server model with the Hazelcast service. This deployment model introduces 
+independent scalability for both servers and clients in Pega Platform. 
+**For Pega Platform deployments using 8.6 and later, this client server is the default and the recommended form of deployment.**
```

Comment:
844-850 includes a redundancy and passive voice...can we simplify?

For Pega Platform deployments using 8.6 and later, Pega recommends adopting a client-server model for your Hazelcast deployment. This deployment model introduces independent scalability for both servers and clients in Pega Platform. To adopt this client-server deployment model, configure the values.yaml section for `hazelcast` to use the Pega-provided `platform/clustering-service` Docker image which contains the Hazelcast clustering service image inside it. Using this image, your deployment starts a cluster of Hazelcast server nodes; a plugin provided by Hazelcast, the Hazelcast-Kubernetes Plugin, discovers the Hazelcast members in the cluster.